src/NexusMods.App.UI/Controls/DevelopmentBuildBanner/DevelopmentBuildBannerView.axaml.cs

```diff
@@ -22,12 +22,11 @@ public DevelopmentBuildBannerView()
         InitializeComponent();
 
         var appVersion = GetAppVersion();
-        Text = $"{appVersion} - DEVELOPMENT USE ONLY";
+        Text = $"{appVersion} - Stardew Valley Preview Build";
     }
 
     private static string GetAppVersion()
     {
-        var prefix = CompileConstants.IsDebug ? "Debug build" : ApplicationConstants.Version.ToString();
-        return $"{prefix} - {ApplicationConstants.CommitHash}";
+        return CompileConstants.IsDebug ? "Debug build" : ApplicationConstants.Version.ToString();
```

Comment:
It should still contain the commit hash if we're in debug mode:

```suggestion
        return CompileConstants.IsDebug
            ? $"Debug build - {ApplicationConstants.CommitHash}"
            : ApplicationConstants.Version.ToString();
```

Reply:
Had to swap some bits around to suit a text change I'm about to commit, the data is still there